test/tools/ossfuzz/protoToYul.h

```diff
@@ -151,6 +165,10 @@ class ProtoConverter
 	unsigned m_numNestedForLoops;
 	// predicate to keep track of for loop init scope
 	bool m_inForInitScope;
+	/// Monotonically increasing counter
+	unsigned m_counter;
```

Comment:
Could they be of type `size_t` as well?

Reply:
I initially made it `size_t` but then realized that the counter (or m_inputSize) can never realistically overflow an unsigned 64 bit number. So I switched to `unsigned` mostly because it does not call attention to itself :-)

Reply:
Alright, I see.